src/app/components/Article/index.stories.jsx

```diff
@@ -1,5 +1,78 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 import Article from './index';
+import { textBlock } from '../../models/blocks';
 
-storiesOf('Article', module).add('default', () => <Article />);
+const title = 'This is a title!';
+const lang = 'en-GB';
+
+const data = {
+  blocks: [
+    {
+      type: 'headline',
+      blockId: '1',
+      model: textBlock('This is a headline!'),
+    },
+    {
+      type: 'text',
+      blockId: '2',
+      model: {
+        blocks: [
+          {
+            blockId: '2-1',
+            type: 'paragraph',
+            model: {
+              text: 'This is some text content!',
+            },
+          },
+          {
+            blockId: '2-2',
+            type: 'paragraph',
+            model: {
+              text: 'More text content!',
+            },
+          },
+        ],
+      },
+    },
+    {
+      type: 'test',
+      blockId: '3',
+      model: {
+        blocks: [
+          {
+            blockId: '3-1',
+            type: 'test-something',
+            model: {
+              text: 'This is some test content!',
+            },
+          },
+        ],
+      },
+    },
+  ],
+};
+
+const filterData = filter => {
```

Comment:
Could this be abstracted out into the helpers as it's likely it will be reused in the future

Reply:
I agree, but let's not abstract it out until there is more use-cases. Needless optimisation otherwise.